manifesto/README.md

```diff
@@ -0,0 +1,8 @@
+# Kyma Elevator Pitch
+
+Kyma is a Cloud Native Application Runtime that gives you the power of Kubernetes but without the hassle of worrying about security, configuration and integration of tools. 
+Kyma does it for you, as it not only combines various components needed for Cloud Native development, but also fills the gaps between them to make them work smoothly together and let you focus on your work. 
+All you need to do is install and set up Kyma and you’re good to go, with a ready-to-use cluster, and a Console UI, too!
+
+The batteries-included approach eliminates the steep learning curve that comes with Kubernetes and ensures easier building and running of microservices and serverless functions and easier service consumption, as we handpick the essential tools from the huge CNCF landscape, glue them together adding whatever’s needed to make them play smoothly with each other, ensure security, top it off with a UI, and wrap in up nicely for you as Kyma so you can jump straight to developing applications.
```

Comment:
```suggestion
The batteries-included approach eliminates the steep learning curve that comes with Kubernetes. It also makes the building and running of microservices and serverless functions easier, and the service consumption with this approach is just a piece of cake. That's why we handpick the essential tools from the huge CNCF landscape, glue them together adding whatever’s needed to make them play smoothly with each other, ensure security, top it off with a UI, and wrap it up nicely for you as Kyma so you can jump straight to developing applications.
```

Reply:
I like the suggestion, thanks! Let's go with that!
It just needs one alteration - `That's why` should actually be `That's because` or something similar.